KeziaA/index.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express')
```

Comment:
The code looks really organized and easy to understand.
- body parameters are only needed in the post route